### DISCUSSION TITLE
Domain ideas

### DISCUSSION THREAD

**Discussion by ebrarkiziloglu** [bounswe/bounswe2024group5#71] [Ideas]
Let's gather our domain idea for our semantic information browser. 
Also let's provide some reason why we think this domain is a good option.

@odenizddd 
@HutkuC 
@semihYILMAZ37 
@set120120  
@SouthAscend 
@Ultiminati 
@ramazanoacar 
@arincdemir

**Comment by SouthAscend** [comment]
I have two similar ideas. The first platform focuses on conspiracy theories. The second platform focuses on fictional content such as movies, books, games, etc.
- A platform about conspiracy theories would be very interesting because most people find them intriguing, either seriously or as a joke. Research can be enjoyable. The final product can become a content haven for influencers and a fun meme source for people who use the app as a joke. It can also be a useful forum for those who take the app seriously.
- A platform for fictional content can be really useful. Similar to Reddit, it can have all sorts of sub-parts like fan art, theories, fun facts, etc. It can also help people with common interests meet each other. Its feed can specialize according to the users' interests. We can link users' accounts with Netflix, IMDb, Letterboxd, goodreads, etc. We can add various original features, like spoiler remover, so that if a new season of a show is out and the user hasn't watched it, there won't be any posts related to it in the user's feed. This project can significantly enhance our capabilities for managing multiple external APIs.

**Comment by arincdemir** [comment]
* Bodybuilding: A reliable platform about bodybuilding is long needed. The internet is bloated with right and wrong statements about bodybuilding. A platform that answers your questions using reliable data sources would be great. In addition to that, being able to interact with real people about this topic would be nice in order to get opinions that are genuine.
* Dancing: Many forums about dancing is outdated or deserted. A place to talk with real people about dancing would be really nice. In addition to that, it is really hard to find content that you are searching for on the internet. There is a lot of content, that's for sure; however, it is hard to find what you are looking for.

**Comment by odenizddd** [comment]
* I think fictional characters would be a good domain. It would be very engaging for users as well as for us, developers. It can also provide people with similar favorite fictional worlds the opportunity to create connections with fellow fans.

**Comment by ebrarkiziloglu** [comment]
My domain idea is outdoor activities. This title involves anything from swimming to mountain biking, or picnicking. Users can post about their activities, adventures, recommend places for specific activities like hiking, share their memories via images, and more. 
I think this topic would be interesting to dive in, considering the fact that most of us, team members, listed some outdoor activities as our hobbies in our profile wiki pages.

**Comment by abdullahumuth** [comment]
My domain idea is university student-related things; student clubs, communities, activities, academic forums, departmental organizations, also for alumni, etc.

The context of my idea is to develop a kind of forum site among university students. (Well, our university has a Facebook group called BUDDY but who uses Facebook anymore? Also ironically, that's what Facebook originally did.)

Bogazici and many other universities have a confession page on Instagram, and that's evolved to be a forum of some sort, usually anonymous. So there should be an anonymous page for that kind of thing as well -but of course where only verified students can post-

We can have a verification system for university students (and that will be integrable to other sites like bouncim etc. similar to login with Google kind of a thing).

**Comment by ramazanoacar** [comment]
My first domain idea is about ai models. It would be easy for developers to share their updates and interact with each other. Also, people with an intention to learn and use some basic models should go over the content in the app.

The other one is similar to other domain suggestions. I think anime could be an interesting domain. Firstly, anime has lots of fans around the world, so gathering some users and letting them discuss or share their thoughts could be a way in our app. Secondly, there are multiple cults in the anime, focusing on them and letting the user keep up with the latest seasons/episodes of some very popular anime could be interesting. In the users perspective, these would be worth to share with each other.

**Comment by HutkuC** [comment]
People should be able to do the following on this platform: ask public or private questions to other users, like a forum, rate songs, albums and artists, personal feed page based on their musical tastes in their profile and music recommendations by the platform. 
The most challenging part of this project will probably be analyzing the data based on users' music tastes and providing similar content.

**Comment by set120120** [comment]
My platform suggestion is about poems. People should get daily poetry suggestions in this app. Must be able to read poems according to subject and interest. Evaluate the "amateur" poems written by other users and make various recommendations. New poem suggestions should be made based on the poems that users like.

**Comment by SouthAscend** [comment]
![WhatsApp Image 2024-03-11 at 17 18 35_d8e9c455](https://github.com/bounswe/bounswe2024group5/assets/137878803/8705416f-b4f0-44dd-b234-68f13b19252a)
We have voted and selected @ebrarkiziloglu's **Outdoor Activites** idea.

**Comment by SouthAscend** [comment]
![image](https://github.com/bounswe/bounswe2024group5/assets/137878803/58e1b27a-965d-4b70-958e-47cf9ae0ad77)
Due to the lack of data in wikidata related to **Outdoor Activites**, we've selected @HutkuC's **Music Platform** idea.
